static/sass/_pattern_search-panel.scss

```diff
@@ -2,8 +2,24 @@
   @include vf-u-truncate;
   @include vf-u-text-muted;
 
+
+  .p-search-roles {
```

Comment:
Seems like there is quite a bit of styles here to put button next to input?

Wouldn't standard Vanilla inline form serve same purpose? https://vanillaframework.io/docs/base/forms#inline

Reply:
The search dropdown doesn't align well if we use grid, because it need to be absolutely positioned. So this alignment doesn't use grid.